src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TraceReporter.kt

```diff
@@ -61,9 +76,11 @@ private fun splitToColumns(nThreads: Int, traceRepresentation: List<TraceEventRe
         // write message in an appropriate column
         result[columnId].add(event.representation)
         val neededSize = result[columnId].size
-        for (column in result)
-            if (column.size != neededSize)
+        for (column in result) {
+            if (column.size != neededSize) {
                 column.add("")
```

Comment:
Coult you please clarify (in code comments) what is going on here? For example, it's unclear why we need to add an empty string.

Reply:
Done